website/src/pages/tutorial.md

```diff
@@ -0,0 +1,530 @@
+## Introduction
+
+In this hands-on tutorial, we will take a look at how we can use Yew to build applications for the browser.
+**Yew** is a modern [Rust](https://www.rust-lang.org/) framework front-end web apps using [WebAssembly](https://webassembly.org/).
+It lets us build web applications in a modern and well-structured way, focusing on the maintainability and re-usability of components
+and management of the application state. It is supported by a large ecosystem of community-created materials and components.
+
+Of course, using webassembly and Rust allows us to reuse our knowledge about language, it's syntax and tooling to
+build front-end applications with modern technology while making use of crates available on [crates.io](https://crates.io/)
+
+### What we are going to build
+
+Rustconf is an intergalactic gathering of the Rust-language community that happens annually.
+Last year, Rustconf 2020 had a plethora of talks that provided a good amount of information.
+In this hands on tutorial, we'll be building a web application that may helps, us, Rustaceans,
+have overview of the talks, and watch them all from one page.
+
+## Setting up
+
+### Prerequisites
+
+To get started, let's make sure we have installed an up-to-date development environment.
+We need the following tools to get started:
+- [Rust](https://www.rust-lang.org/).
+- [`trunk`](https://trunkrs.dev/)
+
+This tutorial also assumes you're already familiar with Rust.
+
+### Setting up the project
+
+First, create a new cargo project:
+
+```bash
+cargo new yew-app
+```
+
+Navigate to the newly created `yew-app` directory. 
+
+```bash
+cd yew-app
+```
+
+To verify the Rust environment is set up properly, run the initial project using the cargo build tool. 
+After output about the build process, you should see the expected "Hello World" message.
+
+```bash
+cargo run
+```
+
+## Our first static page
+
+To convert this simple command line application to a basic Yew web application, a few changes are needed.
+Update the files as follows:
+
+```toml title="Cargo.toml" {7}
+[package]
+name = "yew-app"
+version = "0.1.0"
+edition = "2018"
+
+[dependencies]
++ yew = { git = "https://github.com/yewstack/yew/" }
```

Comment:
Should we recommend using the git version of Yew in the tutorial?
Perhaps there should be something similar on the [Rocket getting started page](https://rocket.rs/v0.5-rc/guide/getting-started/#hello-world)

Reply:
I used git because the there's too many breaking changes between the release version and master. This can be switched to 0.19 once that is released.